### PR TITLE
[ADD] Create maximum_flow_bfs.py

### PR DESCRIPTION
**Pull request by lcw921**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [X] **If creating a new file :**
  - [x] added links to it in the README files ?
  - [X] included tests with it ?
  - [X] added description (overview of algorithm, time and space complexity, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [ ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

**Comment by lcw921** [comment]
implemented the Ford Fulkerson Algorithm algorithm using both bfs and dfs. These are basically the same, but the only difference is whether they use queues or stacks as data structures to find the path.

**Comment by lcw921** [comment]
I implemented the Ford Fulkerson Algorithm algorithm using both bfs and dfs. These are basically the same, but the only difference is whether they use queues or stacks as data structures to find the path.

**Comment by keon** [comment]
please add unittests

**Comment by lcw921** [comment]
Added unit tests. Request you to check.

File: tests/test_graph.py [hunk 1]
```diff
@@ -74,4 +76,36 @@ def test_dijkstra(self):
...
+        unittest {[type]} -- [description]
+    """
+    
+    def test_maximum_flow_dfs(self):
+        graph = [[0, 16, 13, 0, 0, 0], [0, 0, 10, 12, 0, 0], [0, 4, 0, 0, 14, 0], [0, 0, 9, 0, 0, 20], [0, 0, 0, 7, 0, 4], [0, 0, 0, 0, 0, 0]]
+        maximum_flow = maximum_flow_dfs(graph)
+
+        self.assertEqual(maximum_flow, 23)
+
+if __name__ == '__main__':
```
Comment (keon) [inline tests/test_graph.py]: no need to add this here

File: tests/test_graph.py [hunk 2]
```diff
@@ -74,4 +76,36 @@ def test_dijkstra(self):
...
+class TestMaximum_Flow_Dfs(unittest.TestCase):
+
+    """
+    Test for the file def maximum_flow_dfs.py
+    Arguments:
+        unittest {[type]} -- [description]
+    """
+    
+    def test_maximum_flow_dfs(self):
+        graph = [[0, 16, 13, 0, 0, 0], [0, 0, 10, 12, 0, 0], [0, 4, 0, 0, 14, 0], [0, 0, 9, 0, 0, 20], [0, 0, 0, 7, 0, 4], [0, 0, 0, 0, 0, 0]]
```
Comment (keon) [inline tests/test_graph.py]: graph = [
    [0, 16, 13, 0, 0, 0],
    [0, 0, 10, 12, 0, 0],
    [0, 4, 0, 0, 14, 0], 
    [0, 0, 9, 0, 0, 20], 
    [0, 0, 0, 7, 0, 4],
    [0, 0, 0, 0, 0, 0]
]
```

please make it easier to read.
unittests are read by people, and when it breaks, it should be easy to recognize what broke

**Comment by lcw921** [comment]
Plz check it again. I resolve them.

2019년 12월 9일 (월) 오후 5:52, Keon <notifications@github.com>님이 작성:

**Comment by lcw921** [comment]
i resolve them. plz check it again.

**Pull request merged**